### PR TITLE
 🧹  Draft:  Improve azure cnquery packs

### PR DESCRIPTION
**Pull request by HRouhani**
Improvement related to --> https://www.notion.so/mondoo/SOC2-query-pack-required-resources-57920bb45b6a4da7b8f3612cba525849#4dbb1f9d50744f6c8e646f730f9b7f99

All needed resources will be documented at the end of this document: https://www.notion.so/mondoo/Azure-Security-Policy-d1443f97e0bf4634b38bd0b27a0c67da 

Still Ongoing, I believe most of the stuff we can implement.

### CODE REVIEW

File: core/mondoo-azure-inventory.mql.yaml [hunk 1]
```diff
@@ -215,4 +275,10 @@ packs:
         docs:
           desc: |
             This query retrieves data for all resources inside the subscription
-        mql: azure.subscription.resources
\ No newline at end of file
+        mql: azure.subscription.resources
+      - uid: mondoo-asset-inventory-azure-mfa-users
```
Comment (tas50) [inline core/mondoo-azure-inventory.mql.yaml:279]: We need to rework this in a data gathering way, not a security way. We could make it "Retrieve all users in the subscription and their MFA status"

Comment (HRouhani) [reply core/mondoo-azure-inventory.mql.yaml:279]: done, Thanks!

File: core/mondoo-azure-inventory.mql.yaml [hunk 2]
```diff
@@ -23,13 +23,25 @@ packs:
         docs:
           desc: |
             This query retrieves the Azure subscription id
-        mql: azure.subscription.id
+        mql: azure.subscription.subscriptionId
+      - uid: mondoo-asset-inventory-azure-subscription-name
+        title: Retrieve Azure subscription Name
```
Comment (tas50) [inline core/mondoo-azure-inventory.mql.yaml:28]:
```suggestion
        title: Retrieve Azure subscription name
```

**Comment by tas50** [comment]
Closing this out to do it in the new policy instead

**Pull request closed without merging**